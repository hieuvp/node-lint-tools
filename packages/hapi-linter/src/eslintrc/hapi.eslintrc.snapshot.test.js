const hapi = require('./hapi.eslintrc');

it('should match snapshot', () => {
  expect(hapi).toMatchSnapshot();
});
