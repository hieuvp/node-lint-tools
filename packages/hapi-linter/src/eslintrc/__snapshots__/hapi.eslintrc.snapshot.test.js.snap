// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "extends": Array [
    "airbnb-base",
    "plugin:eslint-comments/recommended",
    "plugin:jest/recommended",
    "plugin:jsdoc/recommended",
    "plugin:node/recommended",
    "plugin:prettier/recommended",
    "plugin:promise/recommended",
    "plugin:security/recommended",
    "plugin:unicorn/recommended",
  ],
  "overrides": Array [
    Object {
      "files": Array [
        "runfile.js",
        "*.test.js",
        "*.spec.js",
      ],
      "rules": Object {
        "import/no-extraneous-dependencies": "off",
        "node/no-extraneous-require": "off",
      },
    },
  ],
  "plugins": Array [
    "eslint-comments",
    "jest",
    "jsdoc",
    "node",
    "optimize-regex",
    "prettier",
    "promise",
    "security",
    "unicorn",
    "hapi",
  ],
  "rules": Object {
    "comma-dangle": Array [
      "error",
      "never",
    ],
    "jsdoc/require-param-description": "off",
    "jsdoc/require-returns-description": "off",
    "optimize-regex/optimize-regex": "warn",
    "prettier/prettier": Array [
      "error",
      Object {
        "printWidth": 96,
        "singleQuote": true,
      },
    ],
  },
}
`;
